<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>771</width>
      <height>541</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>5</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Tareas</string>
     </attribute>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>711</width>
        <height>71</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Sitka</family>
        <pointsize>26</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Sistema de Gestión de Tareas</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_14">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>110</y>
        <width>251</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Sitka Display</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Tarea:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lE_tarea">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>140</y>
        <width>271</width>
        <height>41</height>
       </rect>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_tipoTarea">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>230</y>
        <width>231</width>
        <height>41</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>Diaria</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Semanal</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="label_15">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>200</y>
        <width>251</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Sitka Display</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Tipo de tarea:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_16">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>110</y>
        <width>251</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Sitka Display</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Tareas Agregadas</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_17">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>290</y>
        <width>251</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Sitka Display</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Fecha:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lE_infoTarea">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>410</y>
        <width>271</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_18">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>380</y>
        <width>251</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Sitka Display</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Descripcion:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_aceptarTarea">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>400</y>
        <width>171</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>Aceptar</string>
      </property>
     </widget>
     <widget class="QDateEdit" name="fecha_tarea">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>320</y>
        <width>231</width>
        <height>41</height>
       </rect>
      </property>
     </widget>
     <widget class="QTextEdit" name="tE_reporte">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>140</y>
        <width>311</width>
        <height>231</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_5">
     <attribute name="title">
      <string>Figuras</string>
     </attribute>
     <widget class="QGraphicsView" name="graphicsView">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>20</y>
        <width>481</width>
        <height>441</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_triangulo">
      <property name="geometry">
       <rect>
        <x>560</x>
        <y>210</y>
        <width>171</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>Triángulo</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_circulo">
      <property name="geometry">
       <rect>
        <x>560</x>
        <y>120</y>
        <width>171</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>Círculo</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_rectangulo">
      <property name="geometry">
       <rect>
        <x>560</x>
        <y>300</y>
        <width>171</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>Rectángulo</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_19">
      <property name="geometry">
       <rect>
        <x>510</x>
        <y>40</y>
        <width>251</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Sitka Display</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Elija la figura que
 quiera dibujar</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignCenter</set>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Editor de Texto</string>
     </attribute>
     <widget class="QTextEdit" name="tE_editor">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>130</y>
        <width>611</width>
        <height>191</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_bold">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>70</y>
        <width>61</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>B</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_cursive">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>70</y>
        <width>61</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
        <italic>true</italic>
       </font>
      </property>
      <property name="text">
       <string>K</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_underline">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>70</y>
        <width>61</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
        <underline>true</underline>
       </font>
      </property>
      <property name="text">
       <string>U</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Inventario</string>
     </attribute>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>30</y>
        <width>711</width>
        <height>71</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Sitka</family>
        <pointsize>26</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Sistema de Gestión de Inventario</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_agregarM">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>140</y>
        <width>171</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Sitka Display</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Agregar</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_eliminarM">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>230</y>
        <width>171</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Sitka Display</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Eliminar</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_modificarM">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>320</y>
        <width>171</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Sitka Display</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Modificar</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string>Page</string>
     </attribute>
     <widget class="QLineEdit" name="lE_nombreProd1">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>170</y>
        <width>271</width>
        <height>41</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>240</y>
        <width>101</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Sitka Display</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Cantidad:</string>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="dSpinBox_precioProd">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>270</y>
        <width>141</width>
        <height>41</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>240</y>
        <width>151</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Sitka Display</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Precio:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_agregarProd">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>420</y>
        <width>161</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Aceptar</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>140</y>
        <width>251</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Sitka Display</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Nombre del producto:</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="spinBox_cantProd">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>270</y>
        <width>171</width>
        <height>41</height>
       </rect>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_TipoProd">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>170</y>
        <width>171</width>
        <height>41</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>Electronico</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Ropa</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Allimento</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>140</y>
        <width>171</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Sitka Display</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Tipo de producto</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_13">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>30</y>
        <width>711</width>
        <height>71</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Sitka</family>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Sistema de Gestión de Inventario
- Agregar -</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="lbl_new">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>330</y>
        <width>251</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Sitka Display</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Marca:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lE_new">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>360</y>
        <width>231</width>
        <height>41</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="lbl_new2">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>330</y>
        <width>251</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Sitka Display</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Modelo:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lE_new2">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>360</y>
        <width>231</width>
        <height>41</height>
       </rect>
      </property>
     </widget>
     <widget class="QSpinBox" name="spinBox_talla">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>360</y>
        <width>121</width>
        <height>41</height>
       </rect>
      </property>
     </widget>
     <widget class="QDateEdit" name="dateEdit">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>360</y>
        <width>211</width>
        <height>41</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_6">
     <attribute name="title">
      <string>Page</string>
     </attribute>
     <widget class="QLabel" name="label_31">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>30</y>
        <width>711</width>
        <height>71</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Sitka</family>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Sistema de Gestión de Inventario
- Eliminar -</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignCenter</set>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_eliminar">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>150</y>
        <width>321</width>
        <height>121</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_eliminar">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>340</y>
        <width>151</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>PushButton</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_13">
     <attribute name="title">
      <string>Page</string>
     </attribute>
     <widget class="QLabel" name="label_32">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>30</y>
        <width>711</width>
        <height>71</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Sitka</family>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Sistema de Gestión de Inventario
- Modificar -</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignCenter</set>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
